WordPress/src/main/java/org/wordpress/android/ui/stats/service/StatsService.java

```diff
@@ -88,7 +87,11 @@ public int onStartCommand(Intent intent, int flags, int startId) {
             return START_NOT_STICKY;
         }
 
-        final String blogId = StringUtils.notNullStr(intent.getStringExtra(ARG_BLOG_ID));
+        final String blogId = intent.getStringExtra(ARG_BLOG_ID);
+        if (org.apache.commons.lang.StringUtils.isBlank(blogId)) {
```

Comment:
I switched to `isEmpty`, but the semantics of the 2 functions is a little bit different. `isBlank` returns `true` on a string that is a sequence of whitespace characters, but  `isEmpty` doesn't.